addons/sourcemod/scripting/tfgo/event.sp

```diff
@@ -174,8 +173,8 @@ Action Event_ArenaWinPanel(Event event, const char[] name, bool dontBroadcast)
 	{
 		TFGOTeam red = TFGOTeam(TFTeam_Red);
 		TFGOTeam blue = TFGOTeam(TFTeam_Blue);
-		red.PrintToChat("%T", "Team_Cash_Award_no_income_stalemate", LANG_SERVER);
-		blue.PrintToChat("%T", "Team_Cash_Award_no_income_stalemate", LANG_SERVER);
+		red.PrintToChat("%t", "Team_Cash_Award_no_income_stalemate");
+		blue.PrintToChat("%t", "Team_Cash_Award_no_income_stalemate");
```

Comment:
`TFGOTeam.PrintToChat` and `TFGOTeam.AddToClientAccounts` Should format string separately to each clients to print different languages, similar to how `PrintToChatAll` stock is done